api/test/factory/models/oauth2.py

```diff
@@ -33,6 +33,18 @@ class Meta:
     description = Faker("catch_phrase")
     email = Faker("email")
 
+    @factory.post_generation
+    def application(obj, create, extracted, **kwargs):
+        # Only create the application if creating, and either
```

Comment:
```suggestion
        # Only create the application if creating ??, and either
```
If creating `ThrottledApplication` ?

Reply:
If creating the registration. `create` refers to the factory's state: https://factoryboy.readthedocs.io/en/stable/reference.html#postgeneration

> `create` is a boolean indicating which strategy was used